source/gpu_perf_api_common/gpa_hw_info.h

```diff
@@ -230,7 +231,7 @@ class GpaHwInfo
     /// @return True if the current hardware is AMD hardware.
     bool IsAmd() const
     {
-        return vendor_id_set_ && kAmdVendorId == vendor_id_;
+        return vendor_id_set_ && ((kAmdVendorId == vendor_id_) || (kSamsungVendorId == vendor_id_));
     };
```

Comment:
Similar comment here.

Thankfully it doesn't look like we reference these functions in too many places.